### DISCUSSION TITLE
Add option  CPU Temperature Target.

### DISCUSSION THREAD

**Discussion by Matyyy** [seerge/g-helper#3414] [Ideas]
Hi I have a Rog Strix Scar 15 with Intel 12900h and cannot find a CPU Temperature Target slider like the GPU has. I think I have seen some other guys screenshot and they have this option but mine doesn't. I could do this with Throttlestop via PROCHOT Offset but I would like to have this option in G-Helper also.

**Comment by seerge** [comment]
@Matyyy hello, Intel devices don’t support it. But you can just limit CPU power to achieve similar effect. As how temp limit in AMD works - it simply drops power when target temp is reached
